### ISSUE TITLE
SDL1 Support

### ISSUE THREAD

**Issue by RobLoach** [mobius3/KiWi#25]
Is SDL1 support even feasible?

**Comment by mobius3** [comment]
Yeah it would by implementing a render driver for it. It might be a little hackish, but it is feasible.
